CentennialFixups.sln

```diff
@@ -27,86 +27,120 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "MultiByteToWideCharTestFixu
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "FileRedirectionFixup", "fixups\FileRedirectionFixup\FileRedirectionFixup.vcxproj", "{A3653AD0-2406-48A4-95CD-7D4264257F9F}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "PsfShimMonitor", "PsfShimMonitor\PsfShimMonitor.csproj", "{BC76E43C-2E55-4BFD-95DF-DA09884F6DC4}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU

```

Comment:
We should probably hold off on Any CPU configs.  These were removed due to the incomplete support in the PSF for detecting a 64 bit execution of an app from a 32 bit launcher.  In this case, the PSF ends up injecting the 32 bit PSF runtime dll, to host 32 bit shims.  And when the launcher resumes, the target app immediately crashes.  Ultimately, we'd like the PSF to detect the dynamic target app bitness and do the right thing.  But until then, it's best to prevent users easily getting into that situation.

Reply:
The only thing using Any CPU is the PSFMonitor.exe, which is indepdent; it doesn't load any of the  psf dlls.  Any CPU was chosen for that EXE to simplify testing.